src/security/builtin_plugins/access_control/src/access_control_utils.c

```diff
@@ -302,6 +302,8 @@ static bool string_to_properties(const char *str, DDS_Security_PropertySeq *prop
     if (strlen(tok) == 0)
       continue;
     char *name = ddsrt_strsep (&tok, "=");
+    ddsrt_str_trim(name);
```

Comment:
Based on the RFC (if I am reading it correctly), it should really only be trimming space (ASCII 32), not other white space characters. I'm rather doubtful that the `string_to_properties` is correct, given that it doesn't account for escaping, and I am sure this makes it less incorrect.

Am I right in assuming you actually ran into this problem? Because then I think it would make sense to at least do this improvement.